dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -179,11 +179,7 @@ def check_docker_version(verbose: bool):
             text=True,
         )
         if not docker_version_output:
-            console.print(
-                '[red]ERROR[/] Docker version could not be determined. Please make sure Docker is installed '
-                'and running.'
-            )
-            return
+            raise Exception("No running Docker instance found")
```

Comment:
Instead of failing with an ugly-ish traceback, can we catch this in the caller and exit more gracefully instead?

Reply:
Yeah. This is a CLI tool so at any point when we raise exception, it should only happen when there are "unknown" error conditions. 

What we usually do in other parts of the breeze code is:

1) use rich's console (from utils)
2) write the error with `[red[/]" 
3) provide helpful guidance of what to do in `[bright_yellpw][/]" markers
4) sys.exit(1) (or other non-zero error)

I have not documented this yet, but we might want to add ADR for that and do some custom rich "styles" (info/error/guidance etc.) to make it more obvious. I will add an issue for that.

Reply:
Added https://github.com/apache/airflow/issues/22906 to describe the rules (but in the meantime, please follow those and make the message nicer :))

Reply:
Thanks for the information. 
I implemented the changes. We now have a more graceful and helpful message :)

```shell
> python3 dev/breeze/src/airflow_breeze/breeze.py
Docker is not running.
Please make sure Docker is installed and running.
```